test/verify/machineslib.py

```diff
@@ -1345,6 +1345,20 @@ def installWithErrorTest(dialog):
                                              os_name=config.NOVELL_NETWARE_6,
                                              start_vm=False))
 
+            self.machine.execute("cwd=/var/lib/libvirt")
+            server = self.machine.spawn("python3 -m http.server 8080", "httpserver")
+
+            # Test detection of ISO file in URL
+            createTest(TestMachines.VmDialog(self, "subVmTestCreate20", sourceType='url',
+                                             location=config.ISO_URL,
+                                             memory_size=900, memory_size_unit='GiB',
```

Comment:
900GB is too much, it will not return an error because we don't create it, but let's not confuse ourselves.

Reply:
No big deal since this is the only remainder issue.